### ISSUE TITLE
Practice exam 4.C update page 1

### ISSUE THREAD

**Issue by Tonylayton1** [Section508Coordinators/DHSA-TT-200-A#244]
Update the keyboard trap failure to prevent reverse tabbing up and out beyond the About Registration footer link. Implement this the same way as in the incremental. Lastly, add r5e01 to the list of failure reasons for 4.C on this page.

**Comment by mikebarklage** [comment]
Done.

**Comment by Tonylayton1** [comment]
verified on dev

**Comment by drewnielson** [comment]
verified on dev
